modules/ROOT/pages/create-or-modify-a-page.adoc

```diff
@@ -1,65 +1,178 @@
+Forms and pages are made of containers and widgets. +
+Some containers and widgets can be grouped in xref:fragments.adoc[fragments], in order to be reused in several pages or forms. +
+
+The overall graphical structure of the application is handled by the xref:layout-development.adoc[layout]. +
 The look and feel of the application, layout, forms, and pages is handled by the xref:theme-development.adoc[theme].
 
-All of these UI elements are linked to the xref:application-creation.adoc[application descriptor] when the application is created. +
+Forms are embedded in the processes, whereas pages, layouts, and themes are referred to in the application descriptor. +
+
 A default theme is available at Bonita start, both in Bonita Studio and Bonita Runtime. +
-To create a custom theme, developers needs to use their favorite develoment environment and add the theme to the project. +
+To xref:customize-living-application-theme.adoc[create a custom theme], developers needs to use their favorite develoment environment and xref:software-extensibility.adoc[make it available in the project]. +
+
+Bonita UI Designer comes with a set of xref:widgets.adoc[provided containers and widgets]. +
+xref:custom-widgets.adoc[Custom widgets] can also be created to make the palette richer. +
+Those are developped by professional developers, from scratch or from an https://community.bonitasoft.com/project[existing project] of the Community. +
 
 The following chapters give more information on how to develop all the others elements in the UI Designer.
 
-== Design a page, layout, form, or fragment
+== Create a form
+
+In a Bonita project architecture, forms are embedded in the pools. Their creation is very linked to the process step they "feed", in a way. +
+Before creating a form, the xref:contracts-and-contexts.adoc[contract] must have been defined in its _Execution / Contract_ property tab, for each form at a different level: 
+
+* An instantiation form must be created from the pool _Execution / Instatiation form_ tab
+* A task form must be created from the human task _Execution / form_ tab
+
+The Case overview (this page is technically considered as a form because of its strong link to the process) must be created from the pool _Execution / Overview page_ tab and does not need a contract to be created.
+
+. From these tabs, click on the *_Edit_* icon at the right of the _Target form_ field. This opens the UI Designer on a new form, with some widgets, containers, and variables already created to speed up the design process, using the contract inputs.
+. Rename it
+. Customize what needs to be displayed as well as the _Target URL on success_ of the *_Submit_* button.
+
+For more technical information about the forms, visit xref:forms-development.adoc[this dedicated documentation page].
+
+== Create a page
+
+=== From scratch
+. From Bonita Studio coolbar, click on the *_UI Designer_* icon to open the UI Designer. +
```

Comment:
Well this flow is fine, but you can directly create a page from the Studio: New -> page

Reply:
you're right. I'm adding it